File: voteCounting.py
from flask import Flask, request, render_template
import json

app = Flask(__name__)

@app.route('/store_vote', methods=['POST'])
def store_vote():
    data = request.get_json()
    with open('votes.txt', 'a') as file:
        json.dump(data, file)
        file.write('\n')
    return 'Data stored successfully'
@app.route('/')
def home():
    return render_template('index.html')

if __name__ == '__main__':
    app.run()

File: voteCounting.js
const express = require('express');
const app = express();
const fs = require('fs');

app.use(express.static('templates'));

app.post('/store_vote', (req, res) => {
    const data = req.body;
    fs.appendFile('votes.txt', JSON.stringify(data) + '\n', (err) => {
        if (err) throw err;
        console.log('Data stored successfully');
    });
    res.send('Data stored successfully');
});

app.listen(3000, () => {
    console.log('Server started on port 3000');
});
